### PR TITLE
Introduce record_wpr wrapper

### PR DESCRIPTION
**Pull request by atuchin-m**
Resolves https://github.com/brave/brave-browser/issues/40600

The PR:
1. introduces script to record new WPR (or update the current). The upload to s3 still should be done manually.
2. adds more stories to system_health
3. makes the browser language fixed (during the rebase stage)
4. adds `load:site:example:2023` to the regular runs (to replace `about:blank`)

## Submitter Checklist:

- [x] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: tools/perf/brave_page_sets/brave_system_health/system_health_stories.py [hunk 1]
```diff
@@ -104,3 +104,34 @@ class LoadBBCStory2023(_BraveLoadingStory):
...
+class LoadAmazonStory2023(_BraveLoadingStory):
+  NAME = 'load:site:amazon:2023'
+  URL = 'https://www.amazon.com/gp/new-releases/'
+  TAGS = [story_tags.YEAR_2023]
+
+
+class LoadBraveNewsStory2023(_BraveLoadingStory):
+  NAME = 'load:ntp:brave_news:2023'
+  URL = 'chrome://newtab/'
+  TAGS = [story_tags.YEAR_2023]
```
Comment (goodov) [inline tools/perf/brave_page_sets/brave_system_health/system_health_stories.py:137]: why 2023 is used everywhere? What's the point of this tag?

Comment (atuchin-m) [reply tools/perf/brave_page_sets/brave_system_health/system_health_stories.py:137]: Ah, it's chromium originated : https://source.chromium.org/chromium/chromium/src/+/main:tools/perf/page_sets/system_health/story_tags.py;l=66?q=YEAR_2023
Initially, I added 2024, but it cost us +1 patch. Do you thing it makes sense?

Comment (goodov) [reply tools/perf/brave_page_sets/brave_system_health/system_health_stories.py:137]: can we just not use the tag?

Comment (atuchin-m) [reply tools/perf/brave_page_sets/brave_system_health/system_health_stories.py:137]: It requires some non-trivial changes in system_health. Now the tests assume that all the story in the same format.
That approach is also easy to be broken in chrome major bumps.

**Review by goodov — APPROVED** [review]

**Review by iefremov — APPROVED** [review]

**Pull request merged**